model/cloud.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package model
+
+// Product model represents a product on the cloud system.
+type Product struct {
+	ID             string   `json:"id"`
+	Name           string   `json:"name"`
+	Description    string   `json:"description"`
+	DollarsPerSeat float64  `json:"dollars_per_seat"`
```

Comment:
Shouldn't be better PricePerSeat to not couple it to a specific currency?

Reply:
This was inherited directly from CWS definition but I like your naming proposal better. Thanks!